src/irmin-test/store.ml

```diff
@@ -2019,6 +2019,7 @@ let layered_suite (speed, x) =
     match x.layered_store with
     | None -> []
     | Some layered_store ->
+        (Lwt.async_exception_hook := fun exn -> raise exn);
```

Comment:
maybe move this at the top-level of the file ? it's a bit odd to change that global hook inside a function

Reply:
I moved it in the `run` function (initially it was there because only the layered store creates async threads).